server/controllers/allocationsController.ts

```diff
@@ -207,7 +207,16 @@ export default class AllocationsController {
     })
   }
 
-  async getConfirmInstructions(req: Request, res: Response, crn, staffTeamCode, staffCode, convictionNumber, pduCode) {
+  async getConfirmInstructions(
+    req: Request,
+    res: Response,
+    crn,
+    staffTeamCode,
+    staffCode,
+    convictionNumber,
+    pduCode,
+    scrollToBottom = undefined
```

Comment:
Could this be false instead - might be a bit more consistent with the way it is used?!